### DISCUSSION TITLE
Song select insists that a now Ranked song is Qualified, bug?

### DISCUSSION THREAD

**Discussion by DGCK81LNN** [ppy/osu#27950] [Q&A] [answered]
I think I downloaded this song just a short time before it got ranked. I've tried opening the Details view and downloading from the website and manually re-importing it, but the game still insists it's in the Qualified state. Consider this a bug?

I could delete it and download it again, but I don't want it to go all the way to the top when I sort my songs by Date Added.

![osu_2024-04-21_03-23-49](https://github.com/ppy/osu/assets/54282183/50437677-baeb-4a93-80cc-7504ba4fb166)

![osu_2024-04-21_03-23-57](https://github.com/ppy/osu/assets/54282183/72117934-764b-49a0-9289-cba2d4fdfee3)

**Comment by BlockHead007** [comment, marked as answer]
This is known issue: #23804 and #21332 and maybe some more

**Comment by DGCK81LNN** [comment]
ok i see this has been brought up many times. looking forward to a fix. closing this
